scripts/run_nodes.sh

```diff
@@ -193,8 +194,8 @@ function run_node() {
 
 # ------------------------- input checks ----------------------------------
 
-if [[ "${VALIDATORS}" -lt 4 ]]; then
-  error "Number of validators should be at least 4!"
+if [[ "${VALIDATORS}" -lt 1 ]]; then
```

Comment:
Could you print a warning about lack of support for <4? Because I don't think any part of ABFT was written with this possibility in mind, so I wouldn't trust it working consistently with <4 nodes (it does currently afaik, I just don't want to commit to supporting it).

Reply:
I added one anyway.